pkgs/state-proving-manager/src/manager.zig

```diff
@@ -1,15 +1,48 @@
-const types = @import("types")
-const state_transition = @import("state-transition")
+const std = @import("std");
+const ssz = @import("ssz");
+const types = @import("@zeam/types");
+const state_transition = @import("@zeam/state-transition");
+const allocator = std.heap.page_allocator;
```

Comment:
should we be providing in an allocator in the execute_transition fn? in a followup PR